.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
```

Comment:
I think we should also add section for dockerfiles, like in kuma: https://github.com/kumahq/kuma/blob/master/.github/dependabot.yml

Reply:
Ok, it's done